deployment/_shared/scripts/keystore-manager.sh

```
@@ -1389,10 +1428,61 @@ generate_keystore() {
+        DN_PARTS+=("L=$l_val")
+    fi
+
+    # ST — keystore.dn.state
+    local st_val=$(_read_fork_prop "$FORK_PROPS" "keystore.dn.state")
+    if [[ -n "$st_val" ]]; then
+        echo -e "- State (ST): $st_val"
+        DN_PARTS+=("ST=$st_val")
+    fi
+
+    # C — keystore.dn.country (default US)
+    local c_val=$(_read_fork_prop "$FORK_PROPS" "keystore.dn.country")
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)